testutil/testnode/full_node.go

```diff
@@ -135,3 +139,63 @@ func DefaultTendermintConfig() *config.Config {
 	tmCfg.Mempool.MaxTxBytes = 22020096 // 21MB
 	return tmCfg
 }
+
+// DefaultGenesisState will create
+func DefaultGenesisState(fundedAccounts ...string) (map[string]json.RawMessage, keyring.Keyring, error) {
+	encCfg := encoding.MakeConfig(app.ModuleEncodingRegisters...)
+	state := app.ModuleBasics.DefaultGenesis(encCfg.Codec)
+	fundedAccounts = append(fundedAccounts, "validator")
+	kr, bankBals, authAccs := fundKeyringAccounts(encCfg.Codec, fundedAccounts...)
+
+	// set the accounts in the genesis state
+	var authGenState authtypes.GenesisState
+	encCfg.Codec.MustUnmarshalJSON(state[authtypes.ModuleName], &authGenState)
+
+	accounts, err := authtypes.PackAccounts(authAccs)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	authGenState.Accounts = append(authGenState.Accounts, accounts...)
+	state[authtypes.ModuleName] = encCfg.Codec.MustMarshalJSON(&authGenState)
+
+	// set the balances in the genesis state
+	var bankGenState banktypes.GenesisState
+	encCfg.Codec.MustUnmarshalJSON(state[banktypes.ModuleName], &bankGenState)
+
+	bankGenState.Balances = append(bankGenState.Balances, bankBals...)
+	state[banktypes.ModuleName] = encCfg.Codec.MustMarshalJSON(&bankGenState)
+
+	return state, kr, nil
+}
+
+// DefaultNetwork creates an in-process single validator celestia-app network
+// using test friendly defaults. These defaults include fast block times and
+// funded accounts. The returned client.Context has a keyring with all of the
+// funded keys stored in it.
+func DefaultNetwork(t *testing.T, blockTime time.Duration) (cleanup func(), accounts []string, cctx Context) {
+	// we create an arbitrary number of funded accounts
+	for i := 0; i < 300; i++ {
+		accounts = append(accounts, tmrand.Str(9))
+	}
```

Comment:
Append re-allocates memory. So it is more memory performant to initialize the slice and then fill it.
```go
size := 300
accounts = make([]string, 0, size)
for i:=0; i<size; i++ {
  accounts[i] = tmrand.Str(9)
}
```

Reply:
thx for the reminder, we actually have a linter for this but I'm not sure why it didn't get hit